File: Modul-6/Soal-3/PRAK603-2410817220022-AmandaArvaSafaraya.c
#include <stdio.h>

int main() {
    int n1, n2;
    scanf("%d %d", &n1, &n2);

    if (n1 == n2) {
        int matriks[2][n1];
        for (int i = 0; i < 2; ++i) {
            for (int j = 0; j < n1; ++j) {
                scanf("%d", &matriks[i][j]); }
        } 
        for (int j = 0; j < n1; ++j) {
                int hasil = matriks[0][j] * matriks[1][j];
                printf("%d ", hasil); 
            } printf("\n");
    } else {
        printf("Jumlah tidak sama");
    }
    return 0;
}

File: Modul-6/Soal-3/PRAK603-2410817220022-AmandaArvaSafaraya.py
n1, n2 = map(int, input().split())

if n1 == n2:
    matriks = []
    for _ in range(2):
        row = list(map(int, input().split()))
        matriks.append(row)

    hasil = [matriks[0][j] * matriks[1][j] for j in range(n1)]
    print(" ".join(map(str, hasil)))
else:
    print("Jumlah tidak sama")